WeatherToday/Sources/Data/NetworkAPI/RequestConfigurations/WeatherApiRequestConfigurations.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  WeatherApiRequestConfigurations.swift
+//  WeatherToday
+//
+//  Created by Taher on 10/10/22.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Foundation
+import Alamofire
+
+struct WeatherApiRequestConfigurations: RequestConfiguration {
```

Comment:
Optional suggestion: what do you think about converting `WeatherApiRequestConfigurations` from `struct` to `enum`? It will look like:

```suggestion
enum WeatherApiRequestConfigurations: RequestConfiguration {
    case get(cityName: String)
    
    var parameters: Parameters? {
        switch self {
        case let get(cityName): return [
            "q": cityName,
            "appid": Configuration.weatherApiKey,
            "units": "metric"
        ]
        }
    }
}
```

so we can set returning value for `endpoint`, `method`, etc without initialize object.

Reply:
updated in this [commit](https://github.com/nimblehq/ci-cd-sample-ios/pull/20/commits/a14f6fdfc45ee914dd976766a33e952f0c779916)